sdk/ai/azure-ai-projects/samples/inference/sample_chat_completions_with_input_prompt_string.py

```diff
@@ -29,7 +29,7 @@
 def sample_chat_completions_from_input_prompt_string():
     import os
```

Comment:
I love these samples, but not clear to me why they are part of the azure-ai-projects folder and not azure-ai-inference. These samples do not use the AIProjectClient. Do we have the same samples in the azure-ai-projects folder?

Reply:
Updated, the following 2 samples (based on project client) are added:
https://github.com/YusakuNo1/azure-sdk-for-python/blob/users/daviwu/migrate-to-use-prompty/sdk/ai/azure-ai-projects/samples/inference/sample_chat_completions_with_azure_ai_inference_client_and_prompt_string.py
https://github.com/YusakuNo1/azure-sdk-for-python/blob/users/daviwu/migrate-to-use-prompty/sdk/ai/azure-ai-projects/samples/inference/sample_chat_completions_with_azure_ai_inference_client_and_prompty.py